ksqldb-common/src/main/java/io/confluent/ksql/util/KsqlConfig.java

```diff
@@ -509,6 +509,10 @@ String getName() {
   public static final Set<String> SSL_CONFIG_NAMES = sslConfigNames();
   public static final Set<String> STREAM_TOPIC_CONFIG_NAMES = streamTopicConfigNames();
 
+  public static KsqlConfig empty() {
+    return new KsqlConfig(ImmutableMap.of());
```

Comment:
What happened with the static `EMPTY`? That could be reused it, isn't it? Perhaps have the variable back and just return it from this method.

Reply:
It was causing issues because of circular static dependencies. Some things depended on it in a static block and then things wouldn't load. It was very weird...